app/extract_translation_data.py

```diff
@@ -79,6 +79,9 @@ def get_non_question_translatable_text(container):
         extracted_text.extend(get_introduction_translatable_text(container))
     elif container['type'] == 'Interstitial':
         extracted_text.extend(get_interstitial_translatable_text(container))
+    elif container['type'] == 'CalculatedSummary':
+        extracted_text.extend(get_titles_text(container['titles'], container['id']))
+        extracted_text.extend(get_titles_text(container['calculation']['titles'], container['id']))
```

Comment:
Could possibly change this to `get_calculated_summary_text` or something like that.

But works fine and probably not worth changing 👍